vent/api/actions.py

```diff
@@ -176,8 +179,16 @@ def system_commands():
        # restart, shutdown, upgrade, etc.
        return
 
-   @staticmethod
-   def logs():
+   def logs(grep_list=["core","vent-management"]):
```

Comment:
this needs to be tested, I think you're missing `self` as an argument, at a minimum.